nebula/ui/resources/settings-white.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
is this a new file?

Reply:
I couldn't track changes in resources easily.  git mv got confused as I moved things back and forth.  It is needed by nebula.  It came from the old resource qrc.